NAMESPACE

```diff
@@ -53,7 +54,26 @@ export(quickSetupOxcal)
 export(readOxcalOutput)
 export(setOxcalExecutablePath)
 export(wrap_in_boundaries)
+importFrom("ggplot2","aes")
```

Comment:
Clever to import these here. But again: Maybe better to avoid this heavy dependency.

Reply:
See above, results from the doxygen like @imports. Do you think it is save to remove them?

Reply:
If you want to remove them to keep ggplot2 etc. in the Suggests, you have to call each of these functions explicitly in the code, so with their respective namespace (`ggplot2::aes`, etc.).

You should also check if the packages you only have in the Suggests are available at the beginning of the function with something like:

```r
check_if_packages_are_available <- function(packages_ch) {
  if (
    packages_ch %>%
    sapply(function(x) {requireNamespace(x, quietly = TRUE)}) %>%
    all %>% `!`
  ) {
    stop(
      paste0(
        "R packages ",
        paste(packages_ch, collapse = ", "),
        " needed for this function to work. Please install with ",
        "install.packages(c('", paste(packages_ch, collapse = "', '"), "'))"
      ),
      call. = FALSE
    )
  }
}
```

Otherwise the user will get pretty weird feedback.

Reply:
I almost never use the `importFrom` notation, so I don't know how it behaves. But I assume it's the reason why you can't put ggplot2 into the Suggests without triggering
```
Namespace dependencies not required: 'ggplot2', 'ggridges', 'methods'
```
Probably you could go with the `::` notation to avoid this.

I'm not 100% sure if I'm missing something here. You could check our solutions in c14bazAAR and mortAAR for a working example with a lot of the supplementary packages in the Suggests field.

Reply:
With e7e402f now this should be solved?